acceptance/extension-logging-fluentd/package.json

```
@@ -14,7 +14,7 @@
     "directory": "acceptance/extension-logging-fluentd"
   },
   "engines": {
-    "node": "12 || 14 || 16 || 17"
+    "node": "12 || 14 || 16 || >=18"
```

We have been conservative with the engine version range as it's used as a signal for "officially tested & support Node.js versions"

```suggestion
    "node": "12 || 14 || 16 || 17 || 18"
```